app/Http/Controllers/Api/CourseController.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\UserController;
+use App\Models\Course;
+use App\Models\CourseExperiment;
+use App\Models\CourseInstructor;
+use App\Models\CourseResources;
+use App\Models\CourseStudents;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class CourseController extends Controller
+{
+    public function create(Request $request)
+    {
+        $course = new Course();
+
+        $validator = Validator::make($request->all(), [
+            'school_id' => 'required',
+            'faculty_id' => 'required',
+            'title' => 'required',
+            'code' => 'required',
+        ]);
+
+        if ($validator->fails()) {
+            return response()->json(['error' => "All fields are required"], 400);
+        }
+
+        $id = Util::uuid();
+        $school_id = $request->get('school_id');
+        $faculty_id = $request->get('faculty_id');
+        $title = $request->get('title');
+        $code = $request->get('code');
+        $status = $request->get('status') ?? 'Active';
```

Comment:
there is supposed to be an added field called `description`, you seems to have ommited it @bynalab. Can you confirm from the specs/task?